grease-cli/tests/sanity/pass/test-pointer-offset/test.c

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) Galois, Inc. 2024 */
```

Comment:
I would recommend that we write a Crucible S-expression program to test this, rather than checking in a binary. There's nothing binary-specific about what's going on here, and Crucible S-expression files are (1) smaller and (2) easier to diff.

See https://galoisinc.github.io/grease/sexp-progs.html